✨ [RUMF-385] implement a declarative API to set the action names (#412)
DataDog/browser-sdk: packages/rum/README.md

  addUserAction (name: string, context: Context)
  ```

## Declarative API

### Click action naming

> The RUM library will use various strategies to get a name for click actions, but if you want more

Reviewer: ```suggestion
The RUM library is using various strategies to get a name for click actions, but if you want more
```
future tense seems a bit weird to me
